src/GoogleAdWordsEventForwarder.js

```diff
@@ -183,63 +210,33 @@
                 conversionPayload.value = mPEvent.ProductAction.TotalAmount;
             }
 
-            var payload = Object.assign({}, conversionPayload, customProps);
-
-            return payload;
-        }
-
-        // Sends final event to Google or queues if Google isn't ready
-        function sendOrQueueEvent(conversionPayload) {
-            if (window.gtag && forwarderSettings.enableGtag == 'True') {
-                gtag('event', 'conversion', conversionPayload);
-            } else if (window.google_trackConversion) {
-                window.google_trackConversion(conversionPayload);
-            } else {
-                eventQueue.push(conversionPayload);
-            }
+            return mergeObjects(conversionPayload, customProps);
         }
 
-        function logCommerce(event, isPageEvent) {
-            var isPageEvent = false;
-            var conversionLabel = getConversionLabel(event, isPageEvent);
-
-            if (typeof (conversionLabel) !== 'string') {
-                return false;
-            }
-
-            var eventPayload = generateCommerceEvent(event, conversionLabel, isPageEvent);
-
-            if (!eventPayload) {
+        function sendGtagEvent(payload) {
+            try {
+                gtag('event', 'conversion', payload);
+            } catch (e) {
+                console.error('gtag is not available to send payload', payload);
```

Comment:
I think it's helpful to inclue the `e` in the log also. Here and in line 230.  What do you think?